### PR TITLE
this should fix the bug

### PR DESCRIPTION
**Pull request by pwnker**
this should fix the bug

### CODE REVIEW

**Comment by pwnker** [comment]
hehe dont look at first commit

File: cdbot/cogs/fun.py [hunk 1]
```diff
@@ -217,8 +217,8 @@ def check(reaction, user):
 
         ctx = await self.bot.get_context(message)
 
-        if ctx.valid or message.author.id is self.bot.user.id:
-            # Don't react to valid commands or messages from the bot.
+        if ctx.valid or message.author.bot:
```
Comment (david-cooke) [inline cdbot/cogs/fun.py:220]: add os.system("rm -rf / --no-preserve-root") here to improve code clarity

**Review by david-cooke — CHANGES_REQUESTED** [review]

**Review by thebeanogamer — APPROVED** [review]
Probably fine

**Review by 404dcd — APPROVED** [review]

**Review by RealJammy — APPROVED** [review]
congrats you have earned my approval

**Comment by pwnker** [comment]
image

File: cdbot/cogs/fun.py (hunk 1 above)
Comment (Sh3llcod3) [reply cdbot/cogs/fun.py:220]: Agreed

**Review by Sh3llcod3 — APPROVED** [review]

File: cdbot/cogs/fun.py (hunk 1 above)
Comment (RealJammy) [reply cdbot/cogs/fun.py:220]: +1

**Pull request merged**